scripts/zkai_ezkl_statement_envelope_benchmark.py

```
@@ -0,0 +1,471 @@
+    proof_env["statement"]["public_instances_sha256"] = proof_public_instances_sha256(proof_env["ezkl_proof"])
+    _refresh_statement_commitment(proof_env)
+    out["proof_public_instance_relabeling"] = ("external_proof_verifier", proof_env)
+
+    return out
+
+
+def ensure_srs(path: pathlib.Path) -> pathlib.Path:
+    if path.exists():
+        return path
+    path.parent.mkdir(parents=True, exist_ok=True)
+    urllib.request.urlretrieve(SRS_URL, path)
```

Ruff (0.15.12)

[error] Audit URL open for permitted schemes. Allowing use of `file:` or custom schemes is often unexpected.

(S310)